server/index.js

```diff
@@ -439,7 +440,7 @@ app.post('/get_client_urls', function (req, res) {
                 return;
             });
         };
-        
+
         verify_password(body.session, body.password, success, fail);
```

Comment:
Since base32 is case-insensitive, we should make sure all strings are converted to lowercase when saving to the database, as well as comparing with existing db values.